File: ARRAY/1D_Array/OneD_Array.c
#include <stdio.h>

int main() {
    int size;
    printf("Enter the size of the array: ");
    scanf("%d", &size);  // User inputs the size of the array

    int arr[size];  // Declare array dynamically

    // Taking input from the user
    printf("Enter %d elements:\n", size);
    for (int i = 0; i < size; i++) {
        scanf("%d", &arr[i]);
    }

    // Display the elements
    printf("Array elements are: ");
    for (int i = 0; i < size; i++) {
        printf("%d ", arr[i]);
    }
    
    return 0;
}

File: ARRAY/1D_Array/OneD_Array.java
import java.util.Scanner;

public class OneD_Array {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter the size of the array: ");
        int size = sc.nextInt();

        // Create an array of given size
        
        int[] arr = new int[size];  
        
        System.out.println("Enter " + size + " elements:");
        for (int i = 0; i < size; i++) {
            arr[i] = sc.nextInt();  // Input elements from user
        }
        
        System.out.println("Array elements are:");
        for (int num : arr) {
            System.out.print(num + " ");
        }
        
        sc.close();
    }
}
